src/core/transpile.jl

```diff
@@ -2447,3 +2447,52 @@ function transpile(
 
     return circuit
 end
+
+struct RejectGatesOnExcludedPositionsTranspiler <: Transpiler
```

Comment:
Please see other transpilers for how to add documentation and doctests to public functions.

Reply:
Documentation and doctests have been added.